config.json

```diff
@@ -2,6 +2,7 @@
   "language": "Groovy",
   "checklist_issue": 9,
   "active": false,
+  "blurb": "Apache Groovy is an optionally typed, optionally dynamic, object-oriented and functional language which aims to improve developer productivity and happiness.  It is fully compatible with the JVM ecosystem.",
```

Comment:
If possible, let's see if we can get something more conversational here. The "optionally typed / optionally dynamic" and "object-oriented and functional" make this pretty difficult to grasp.

Here are three blurbs that I find both conversational and intriguing:

The R blurb reads:

> R is an interpreted, open source programming language with powerful data manipulation and graphical capabilities. It has a thriving package ecosystem and is one of the most popular languages for data science.

The Ruby blurb reads:

> Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.

The Go one reads:

> Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library, and fantastic tooling. It's a great fit for web backends and command-line tools.

Some questions that might help find the tone and contents for the Groovy blurb.

* What is Groovy typically used for?
* What about the language makes people happy? (In Ruby it's _a syntax that is natural to read and easy to write_, whereas for Go it's _small, consistent syntax_).
* What pain points was Groovy specifically trying to address when the creators started working on it?
* What do you like about writing Groovy?
* If you were out having a drink with a programmer friend who had never written Groovy, what would you tell them to get them interested in the language?

Feel free to riff on these in verbose ways, we can winnow down once we find a direction.

Reply:
@kytrinyx Thank you so much!  I'm wowed by how awesome and insightful that post is— and thank you for taking the time.

I'm going to reflect for a while before blurting out my thoughts on the great questions above.  (I'd love to hear @m-a-ge 's thoughts, as well.)

Reply:
@kytrinyx, thanks for your explanation. The description is a bit formal, we could come with something else.

For me **Groovy** is an unopinionated multi-paradigm language which enhances Java and makes you code less verbose. It made me happy working with `xml` (could you imagine that?) and I love its `groovyConsole` (similar to Python`s `IDLE`) which allows you to try out some code snippet very fast (in comparison with Java, but I guess Java 9 somewhat solve this). You can also write Groovy scripts (like in Python) which is awesome!
Groovy is an open source language which is totally dependent on JVM, we should mentioned that as well.

@Dispader, let's come up with our own description, what do you think? 

P.S. Sorry for the delay, my arm is finally healed 🙃

Reply:
> For me Groovy is an unopinionated multi-paradigm language which enhances Java and makes you code less verbose.

This is great.

> It made me happy working with xml (could you imagine that?)

Haha, only barely!

> let's come up with our own description

🎉